Common/Securities/SymbolPropertiesDatabase.cs

```diff
@@ -35,6 +35,34 @@ private SymbolPropertiesDatabase(IReadOnlyDictionary<SecurityDatabaseKey, Symbol
         {
             _entries = entries.ToDictionary();
         }
+        
+        /// <summary>
+        /// Check whether symbol properties exists for the specified market/symbol/security-type
+        /// </summary>
+        /// <param name="market">The market the exchange resides in, i.e, 'usa', 'fxcm', ect...</param>
+        /// <param name="symbol">The particular symbol being traded</param>
+        /// <param name="securityType">The security type of the symbol</param>
+        public bool ContainsKey(string market, string symbol, SecurityType securityType)
+        {
+            var key = new SecurityDatabaseKey(market, symbol, securityType);
+            return _entries.ContainsKey(key);
+        }
+
+        /// <summary>
+        /// Check whether symbol properties exists for the specified market/symbol/security-type
+        /// </summary>
+        /// <param name="market">The market the exchange resides in, i.e, 'usa', 'fxcm', ect...</param>
+        /// <param name="symbol">The particular symbol being traded (Symbol class)</param>
+        /// <param name="securityType">The security type of the symbol</param>
+        public bool ContainsKey(string market, Symbol symbol, SecurityType securityType)
+        {
+            var stringSymbol = symbol == null ? string.Empty :
```

Comment:
There is a `public static MarketHoursDatabase.GetDatabaseSymbolKey()` that handles this logic, maybe it would be better to directly call it instead